src/main/java/com/grimbo/chipped/compat/ChippedRecipeCategory.java

```diff
@@ -29,13 +30,13 @@
     private final ItemStack icon;
 
 	public ChippedRecipeCategory(Block block) {
-		id = CategoryIdentifier.of(Registry.BLOCK.getKey(block));
```

Comment:
Fix the formatting here.

Reply:
that's odd, it doesnt show up on intellij

Reply:
Could just be github being weird lol